distribution/docs/src/main/resources/_contents/_web-applications/admin-console-contents.adoc

```diff
@@ -0,0 +1,162 @@
+
+=== ${admin-console}
+
+The ${admin-console} is the centralized location for administering the system.
+The ${admin-console} allows an administrator to install and remove selected applications and their dependencies and access configuration pages to configure and tailor system services and properties.
+The default address for the ${admin-console} is \${secure_url}/admin.
+
+==== ${admin-console} Modules
+
+The ${admin-console} is a modular system that can be expanded with additional modules as necessary.
+${ddf-admin} comes with the Configuration module and the Installer modules.
+However, new modules can be added, and each module is presented in its own tab of the ${admin-console}.
+
+Admin Modules are single components that implement the `org.codice.ddf.ui.admin.api.module.AdminModule` interface.
+Once they implement and expose themselves as a service, they are added to the ${admin-console} as a new tab.
+
+===== Included Modules
+
+* <<_installer_module, Installer Module>>
+* <<_configuration_module, Configuration Module>>
+
+====== Installer Module
```

Comment:
Yeah. it's weird, but that's how the default stylesheet does `<h6>` section headings